web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -24,6 +24,10 @@
 User = get_user_model()
 
 
+HOUR_MINUTE_FORMAT = "%I:%M"
+MONTH_DAY_FORMAT = "%m:%d"
```

Comment:
The `:` notation on the month day format is a little bit confusing. why not display full date including `%Y`?

Reply:
The requirements for the ticket specified this display. The ":" is a literal colon, not notation.

Reply:
Thanks for the correction. I can't find `MONTH_DAY_FORMAT = "%m:%d"` on the ticket description. This makes it hard to differentiate time from the date. example: `at 10:30 on
07:10 (America/New_York)`